Rakefile

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
+
+RSpec::Core::RakeTask.new(:test)
```

Comment:
in general, instead of 'bundle exec rake spec` or `bundle exec rake test` it's now preferable to run with `bundle exec rspec`. The rspec runner has become very powerful over the last year, and exposes much better command line flags than the rake task.

Reply:
I'll make sure to do that locally, but in terms of running the CI I'm going to be coordinating everything through the rake task, so it's still useful to have this.